v4/source/cloud.yaml

```
@@ -342,3 +342,26 @@
                 $ref: "#/components/schemas/SubscriptionStats"
         "500":
           $ref: "#/components/responses/InternalServerError"
+
+  /subscription/limitreached/invite:
+    post:
+      tags:
+        - cloud
+      summary: POST endpoint for triggering sending emails to admin with request to upgrade workspace
+      description: >
+        An endpoint that triggers sending emails to all sys admins to request them to upgrade the workspace
+        ##### Permissions
+
+        This endpoint should only be accessed in a Mattermost Cloud instance
+
+        __Minimum server version__: 5.34
+        __Note:__ This is intended for internal use and is subject to change.
+      responses:
+        "200":
+          description: Atleast one sys admin received the an email
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
```

This should be set back to 413 as originally implemented

```suggestion
        "413":
          $ref: "#/components/responses/TooLarge"
```